tools/targets.py

```diff
@@ -497,6 +497,13 @@ def ncs36510_addfib(t_self, resources, elf, binf):
         from tools.add_fib import add_fib_at_start
         print("binf ", binf)
         add_fib_at_start(binf[:-4])
+
+class RTL8195ACode:
+    """RTL8195A Hooks"""
+    @staticmethod
+    def binary_hook(t_self, resources, elf, binf):
+        from tools.build_hooks.REALTEK_RTL8195AM import rtl8195a_elf2bin
```

Comment:
Also, these changes will have to go to `tools/targets/__init__.py`, but just doing a rebase of your branch onto the current master should resolve that automatically.